biochemist-python/chapters/Nonlinear Curve Fitting Part 1.ipynb

```diff
@@ -0,0 +1,1436 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Inspect the data using the plot command that is available with the dataframe\n",
+    "AP_kinetics_df_transpose.plot(marker = 'o')"
```

Comment:
We don't cover this plot command specifically - it is a different type of plotting than covered in the plotting lesson. We need to be consistent with this - should either use plotting introduced in the plotting lesson or should explicitly discuss this type of plot as part of the lesson.

Reply:
The pyplot command to plot all of those curves is going to be pretty involved. this is what I wrote about using the pandas dataframe.plot function. Is this sufficient?

Inspect the data

In an earlier module, we used pyplot from matplotlib to create a well-annotated plot of our linear regression data. We could do that here, but we only want to inspect the data to make sure we are on the right track. To do that we can use the plot command from pandas, which builds the plot using tools from matplotlib. The syntax is

dataframe.plot()

In our case, the only argument we will pass is "marker = 'o' so that the individual data points will appear.